src/constants.js

```diff
@@ -38,5 +58,14 @@ module.exports = {
   updateActions: {
     MOVE: 'move',
     CHANGE_COORDINATES: 'change_coordinates'
+  },
+  meta: {
+    FEATURE: 'feature',
+    MIDPOINT: 'midpoint',
+    VERTEX: 'vertex'
+  },
+  activeStates: {
+    ACTIVE: 'true',
+    INACTIVE: 'false'
   }
```

Comment:
It would be nice to know that `meta` and `activeStates` were related to render features.